ISLE/isleapp.cpp

```diff
@@ -89,6 +91,21 @@ SDL_Window* window;
 
 extern const char* g_files[46];
 
+#ifdef __EMSCRIPTEN__
```

Comment:
I'd say let's leave out Fullscreen support until they fixed the bug in SDL. I'd rather not have this hack and I think we can go without FS for now (you can have "windowed" Fullscreen anyway with just using CSS)